packages/backend/src/migration/1593764167709-alterTables.ts

```diff
@@ -4,18 +4,69 @@ export class alterTables1593764167709 implements MigrationInterface {
     public async up(queryRunner: QueryRunner): Promise<void> {
+
+        await queryRunner.query(`ALTER TABLE "modos"."profile" RENAME COLUMN "helperFrequency" to "helper_frequency";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."user" RENAME COLUMN "passwordHash" to "password_hash";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."user" RENAME COLUMN "profileId" to "profile_id";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."user" DROP CONSTRAINT IF EXISTS "REL_9466682df91534dd95e4dbaa61";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."user" ADD CONSTRAINT " user_profileid_unique" UNIQUE ("profile_id");`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "ownerId" to "owner_id";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "eventId" to "event_id";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "descriptionObstacle" to "description_obstacle";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "descriptionFreetext" to "description_freetext";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "descriptionImpact" to "description_impact";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "imageBasename" to "image_basename";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "imageWidth" to "image_width";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "imageHeight" to "image_height";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "locationLatitude" to "location_latitude";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "locationLongitude" to "location_longitude";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" RENAME COLUMN "locationAltitude" to "location_altitude";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."event" RENAME COLUMN "numberOfImages" to "number_of_images";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."event" RENAME COLUMN "ownerId" to "owner_id";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."user_events_event" RENAME COLUMN "userId" to "user_id";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."user_events_event" RENAME COLUMN "eventId" to "event_id";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."user_events_event" DROP CONSTRAINT IF EXISTS "PK_11948eb9a443f34df93cac35feb";`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."user_events_event" ADD CONSTRAINT "user_events_unique" PRIMARY KEY ("user_id", "event_id");`, undefined);
```

Comment:
If you change that without also changing the models in typeorm, nothing will work anymore...

Reply:
(So please do not change that)

Reply:
Remove all Renaming Thanks you !